spring-boot-dm/camel-spring-boot-generator-bom/pom.xml

```diff
@@ -209,9 +209,15 @@
                             <exclude>de.flapdoodle*:*</exclude>
                             <exclude>org.apache.activemq:*</exclude>
                             <exclude>org.apache.commons:commons-pool2</exclude>
+                            <exclude>org.apache.httpcomponents:httpasyncclient</exclude>
+                            <exclude>org.apache.httpcomponents:httpclient</exclude>
+                            <exclude>org.apache.httpcomponents:httpclient-cache</exclude>
+                            <exclude>org.apache.httpcomponents:httpclient-osgi</exclude>
                             <exclude>org.apache.httpcomponents:httpcore</exclude>
+                            <exclude>org.apache.httpcomponents:httpmime</exclude>
                             <exclude>org.apache.derby:derby</exclude>
                             <exclude>org.apache.logging.log4j:*</exclude>
+                            <exclude>org.cassandraunit:*</exclude>
```

Comment:
@nicolaferraro is this correct, should cassandraunit be exlucded in the SB bom generator?